docs/docs/misc/migration_notes.md

```diff
@@ -8,6 +8,29 @@ Aztec is in full-speed development. Literally every version breaks compatibility
 
 ## 0.36.0
 
+## [Aztec.nr & js] Portal addresses
+
+Deployments have been modified. No longer are portal addresses treated as a special class, being immutably set on creation of a contract. They are no longer passed in differently compared to the other variables and instead should be implemented using usual storage by those who require it. One should use the storage that matches the usecase - likely shared storage to support private and public.
+
+This means that you will likely add the portal as a constructor argument
```

Comment:
I'd add this here:

```diff
struct Storage {
+   portal_address: SharedImmutable<AztecAddress>,
}
```

Can't do a proper gh suggestion since the code formatting screws up with the suggestion formatting :facepalm:

Reply:
If done in GH, you can write "suggestion" as the language.